examples/flutter_gallery/meta/flutter_gallery.cmx

```diff
@@ -4,7 +4,7 @@
     },
     "sandbox": {
         "system": [ "data" ],
-        "features": [ "persistent-storage" ],
+        "features": [ "isolated-persistent-storage" ],
```

Comment:
It looks like this file may have been copied from somewhere else without thinking about what was actually needed here. I suspect that flutter_gallery doesn't need persistent storage, and so this line can be deleted. The previous line ("system": ["data"]) can probably also be deleted. The "services" can almost certainly be pruned down as well since flutter_gallery doesn't explicitly use any FIDL services, but maybe those can be pruned at a later time.

Reply:
Done